### PR TITLE
Use auth-json-server for testing and some housekeeping

### PR DESCRIPTION
**Pull request by eliangcs**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // You'll want to set these with either `CLIENT_ID=abc zapier test` or `zapier env 1.0.0 CLIENT_ID abc`
-process.env.BASE_URL = process.env.BASE_URL || 'https://oauth-test.zapier.com/oauth';
+process.env.BASE_URL = process.env.BASE_URL || 'https://auth-json-server.zapier.ninja';
```
Comment (FokkeZB) [inline index.js:2]: Shouldn't this use a public host so that users can run the tests?

**Review by FokkeZB — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (eliangcs) [reply index.js:2]: https://auth-json-server.zapier.ninja is a public host.

Comment (FokkeZB) [reply index.js:2]: O wow! I thought we reserved `.ninja` to the staging environment only available via VPN. Learned something today :)

**Review by FokkeZB — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (eliangcs) [reply index.js:2]: That was what I thought, too, until devops told me otherwise. :)

**Pull request merged**